wemake_python_styleguide/violations/consistency.py

```diff
@@ -438,17 +438,18 @@ class MultipleInComparisonViolation(ASTViolation):
         Bring a consistency to the comparison!
 
     Solution:
-        Refactor your comparison expression.
-        Use different Comparisons.
+        Refactor your comparison expression to use several `and` conditions
+        or separate `if` statements in case it is appropriate.
 
     Example::
 
         # Correct:
-        if item in list:
-        if 1 in [3,4]:
+        if item in list and list in master_list:
```

Comment:
You can not name your variables as `list`. It is a builtin. Please, change the name.